src/blocks/block-layout/components/layout/layout-library-item-card.js

```diff
@@ -37,12 +36,10 @@ export default class LayoutLibraryItemCard extends Component {
 									this.props.import( this.props.content );
 								} }
 							>
-								<LazyLoad>
```

Comment:
Is this `<LazyLoad>` important? If so, I'll add it back in and ignore the console warning it raised.

It looks to have caused:
>Warning:componentWillReceiveProps has been renamed,

https://github.com/loktar00/react-lazy-load/issues/135

Reply:
@kienstra Looks like `LazyLoad` was also removed and merged in this recent PR: https://github.com/studiopress/atomic-blocks/pull/368/files

Reply:
Oh, thanks for pointing that out. I should also pull in the latest from `develop`